src/AutoMapper/ResolutionContext.cs

```diff
@@ -10,33 +10,36 @@ public class ResolutionContext : IInternalRuntimeMapper
     {
         private Dictionary<ContextCacheKey, object> _instanceCache;
         private Dictionary<TypePair, int> _typeDepth;
-        private readonly IInternalRuntimeMapper _inner;
+        private readonly IInternalRuntimeMapper _mapper;
         internal ResolutionContext(IMappingOperationOptions options, IInternalRuntimeMapper mapper)
         {
             Options = options;
-            _inner = mapper;
+            _mapper = mapper;
         }
-        internal ResolutionContext(IInternalRuntimeMapper mapper) : this(mapper.DefaultContext.Options, mapper) { }
+        internal ResolutionContext(IInternalRuntimeMapper mapper) : this(null, mapper) { }
         /// <summary>
         /// Mapping operation options
         /// </summary>
-        public IMappingOperationOptions Options { get; }
+        internal IMappingOperationOptions Options { get; }
         /// <summary>
         /// Context items from <see cref="Options"/>
         /// </summary>
         public IDictionary<string, object> Items
         {
             get
             {
-                CheckDefault();
+                if (Options == null)
+                {
+                    ThrowInvalidMap();
```

Comment:
We could simply return null here but I'm guessing people would keep asking why `Items` is null.